comps/agent/langchain/src/agent.py

```diff
@@ -1,30 +1,37 @@
 # Copyright (C) 2024 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
+from .utils import load_python_prompt
 
 
 def instantiate_agent(args, strategy="react_langchain", with_memory=False):
+    if args.custom_prompt is not None:
+        print(f">>>>>> custom_prompt enabled, {args.custom_prompt}")
+        custom_prompt = load_python_prompt(args.custom_prompt)
```

Comment:
@minmin, this function will be used here, it is transparent to strategy developers